src/features/farming/animals/components/MutantAnimalModal.tsx

```diff
@@ -85,24 +21,52 @@ interface Props {
 
 export const MutantAnimalModal = ({ mutant, show, onContinue }: Props) => {
   const { t } = useAppTranslation();
+  const boost = COLLECTIBLE_BUFF_LABELS[mutant];
+  const isPermanentMutant = (name: string): boolean => {
+    const permanentMutants = ["Speed Chicken", "Rich Chicken", "Fat Chicken"];
+    return permanentMutants.includes(name);
+  };
```

Comment:
Speed, Rich and Fat chicken are no longer able to be dropped from chickens anymore, so they aren't permanent anymore